program/index.html

```diff
@@ -197,6 +209,26 @@ <h2 style="text-align:center">Monday March 28</h2>
 	<td colspan="2">Welcome and plenary</td>
 </tr>
 
+<tr>
+    <td></td>
+    <td class="pd">1:00</td>
+    <td><i>Organizers</i></td>
+    <td>Housekeeping</td>
+</tr>
+
+<tr>
+    <td></td>
+    <td class="pd">1:05</td>
+    <td><i>John Todd</i></td>
+    <td>About the Welcome Centre for Human Genetics (WCHG) and speaker introduction</td>
+</tr>
+
+<tr>
+    <td></td>
+    <td class="pd">1:15</td>
+    <td><i>Mara Lawniczak</i></td>
+    <td>Plenary title</td>
```

Comment:
How about "Title TBC"? We want to merge ASAP and can update as we get the details. Don't want this diff lying around for days.